_layouts/post.html

```diff
@@ -1,58 +1,66 @@
-                Updated {{ page.date | date_to_string }}
-            </p>
-            {% if page.tags %}
-              {% for tag in page.tags %}
-                <span class="badge badge-pill badge-default">
-                  {% capture tag-name %}{{tag | upcase }}{% endcapture %}
-                  {% if tag-name=="AWS" or tag-name=="GCP"  or tag-name=="GKE" %} {{ tag-name }} {% else %} {{tag-name | capitalize}}{% endif %}</span>
-              {% endfor %}
-            {% endif %}
-          </div>
-          <div class="content">
-              {{ content }}
-          </div>
-
-    </div>
-    <div class="col-md-2">
-        <div class="card contact-us-card text-center">
-          <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
-            <div class="card-body">
-              <p class="card-title">Want to learn more? Feel free to contact us for support</p>
-              <a href="#" class="btn btn-primary">Support</a>
-              <a href="#">Dismiss</a>
-            </div>
-          </div>
     </div>
+    <div class="col-md-7 guides-section-white">
+      <ol class="breadcrumb text-center">
+        {% assign crumbs = page.url | split: '/' %}
+        <a href="/guides" ga-event-category="guide-{{ post.title | downcase}}" ga-event-action="deployment-guides">Dev Ops Guides</a> 
+        {% for crumb in crumbs offset: 2 %}
+          {% if forloop.last %}
+           <a href="#"> / {{ crumb | replace:'-',' ' | capitalize }}</a>
+          {% else %}
+            <a href="{{ crumb }}">/ {{ crumb | replace:'-',' ' | capitalize }}</a>
+          {% endif %}
+        {% endfor %}      
+      </ol>
+      <div class="post-title text-center">
+        <img class="img-center" src="{{ page.image }}" alt="post logo">
+        <label>{{ page.title }}</label>
+        <p>
+          {% assign words = content | number_of_words %}
+          <!-- https://help.medium.com/hc/en-us/articles/214991667-Read-time -->
+          {% if words < 265 %}
+            1 min read.
+          {% else %}
+            {{ words | divided_by:180 }} mins read.
+          {% endif %} 
+          Updated {{ page.date | date_to_string }}
+        </p>
+        {% if page.tags %}
+          {% for tag in page.tags %}
+            <span class="badge badge-pill badge-default">
+              {% capture tag-name %}{{tag | upcase }}{% endcapture %}
+              {% if tag-name=="AWS" or tag-name=="GCP"  or tag-name=="GKE" %} {{ tag-name }} {% else %} {{tag-name | capitalize}}{% endif %}</span>
+          {% endfor %}
+        {% endif %}
+      </div>
+      <div class="content">
+          {{ content }}
+      </div>
     </div>
+    <div class="col-sm-2-5">
+      <div class="card guides-cta-card text-center">
+        <img src="/assets/img/grunty-cta.png" alt="newsletter_image" class="img-center"><br>
+        <div class="card-body">
+          <p class="card-title">Hand off the gruntwork, we can deploy this service for you. </p>
```

Comment:
There are many differences between this element and it's design.

1. Position of grunty (should be higher)
2. Quality of grunty image (should be higher)
3. Width of box (should be narrower)
3. Box shadow (should be more difused)
4. Font weight ((should be thicker)
5. Font color (should be gray, not black)
6. Button color (different shade of blue)


Expected:
<img width="275" alt="Screen Shot 2019-09-05 at 14 08 41" src="https://user-images.githubusercontent.com/1516418/64349536-0d5f6600-cfef-11e9-979e-7cb7ac71dec5.png">
Current: 

<img width="345" alt="Screen Shot 2019-09-05 at 14 08 51" src="https://user-images.githubusercontent.com/1516418/64349550-13eddd80-cfef-11e9-9976-1e46b3c75871.png">

Reply:
I didn't continue with this because there was still discussion around if it should be taken off or not. Jim confirmed @ No 8 that this should be removed

Reply:
Ok!